NebulaHost/MultiplayerHostSession.cs

```diff
@@ -49,7 +50,10 @@ public void StartServer(int port)
             LocalPlayer.IsMasterClient = true;
 
             // TODO: Load saved player info here
-            LocalPlayer.SetPlayerData(new PlayerData(PlayerManager.GetNextAvailablePlayerId(), GameMain.localPlanet?.id ?? -1, new Float3(1.0f, 0.6846404f, 0.243137181f)));
+            if (!isDedicated)
+            {
+                LocalPlayer.SetPlayerData(new PlayerData(PlayerManager.GetNextAvailablePlayerId(), GameMain.localPlanet?.id ?? -1, new Float3(1.0f, 0.6846404f, 0.243137181f)));
+            }
```

Comment:
I think that we should still do the `LocalPlayer.SetPlayerData` even if the dedicated host does not have an actual player. Else the host and the first player that joins will have the same player id and it could cause some issues in the future.